src/utils/get.ts

```diff
@@ -0,0 +1,13 @@
+import isUndefined from './isUndefined';
+import isNullOrUndefined from './isNullOrUndefined';
+
+export default (obj: any, path: string, defaultValue?: any) => {
+  const result = path
+    .split(/[,[\].]+?/)
+    .filter(Boolean)
```

Comment:
I think this `filter` is unnecessary.
May I create PR?

Reply:
sure

Reply:
Sorry, I misunderstood!
This `filter` is necessary.